pkg/controller/workload/kubernetes/resource/resource.go

```diff
@@ -267,7 +267,8 @@ func (c *unstructuredClient) sync(ctx context.Context, template *unstructured.Un
 
 	rs := getRemoteStatus(remote)
 
-	return rs, errors.Wrap(resource.Apply(ctx, c.kube, template), errSyncResource)
+	// TODO(negz): Update to resource.APIPatchingApplicator.
+	return rs, errors.Wrap(resource.Apply(ctx, c.kube, template), errSyncResource) // nolint:staticcheck
```

Comment:
Just a heads up I believe this change is also made in https://github.com/crossplane/crossplane/pull/1409 but shouldn't matter which comes through first 👍